packages/scenes/src/components/layout/grid/SceneGridLayout.tsx

```diff
@@ -14,6 +14,7 @@ interface SceneGridLayoutState extends SceneObjectState {
    * Turn on or off dragging for all items. Indiviadual items can still disabled via isDraggable property
    **/
   isDraggable?: boolean;
+  lazy?: boolean;
```

Comment:
```suggestion
  isLazy?: boolean;
```

just to keep same naming as isDraggable